src/api/Intelligence.js

```diff
@@ -25,12 +24,9 @@ class Intelligence extends Base {
      *
      * @param {string} prompt - Question
      * @param {Array<object>} items - Array of items to ask about
-     * @param {Function} successCallback - Function to call with results
```

Comment:
should the property defined in L12-L20 be deleted as well

Reply:
deleted